pkg/cmdutils/types.go

```diff
@@ -98,6 +98,12 @@ type ExecuteArgs struct {
 	DefaultAppName string
 }
 
+type DiagramCmd struct {
+	IntegrationValue      string
```

Comment:
what's integration and sequence value?

Reply:
That will hold the app name and endpoints necessary for a that diagram. In case of endpoint analysis, it doesn’t need anything other than the module. In order to differentiate, I have added a bool to it which needs to be set to true. That’s the only way I could think of to have all three under the same command.

Reply:
Alright will do!